pup.py

```diff
@@ -62,8 +62,38 @@
     await asyncio.sleep(0.1)
 
 
-async def handle_file(msgs):
+def fail_upload(data, result):
```

Comment:
Since we're passing in `response`, do you think we should rename the `result` parameter to `response` just to keep things named the same?